app/views/partials/droits-eligibles-list.html

```diff
@@ -1,10 +1,9 @@
-<accordion close-others="true">
-  <accordion-group is-open="droit.open"
-               class="resultats-accordion"
-               ng-repeat="(droitId, droit) in list"
-               ng-class="{'resultats-accordion-needs-n-2': droit.isBaseRessourcesYearMoins2 && !ressourcesYearMoins2Captured && ! isString(droit.montant), 'panel-opened': droit.open}">
+<uib-accordion close-others="true">
+  <uib-accordion-group is-open="droit.open"
+    class="resultats-accordion {{ getAccordionClass(droit) }}"
```

Comment:
:heart: for forward comment  :wink:

Might benefit from a comment though, if possible with a reference to the doc that explains why.